go.mod

```diff
@@ -37,10 +38,11 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/gokyle/hotp v0.0.0-20160218004637-c180d57d286b
 	github.com/golang/protobuf v1.5.2
-	github.com/google/btree v1.0.0
+	github.com/google/btree v1.0.1
 	github.com/google/go-cmp v0.5.6
 	github.com/google/gops v0.3.14
 	github.com/google/uuid v1.2.0
+	github.com/gorilla/websocket v1.4.2
```

Comment:
I just talked to Alexey about it and while gorilla is more maintained I'll be switching to the `x/net` package.

Reply:
I made a late call and talked with Alexey to switch over to Gorilla when I am already working on this. It'll clear up some technical debt and a bunch of code that's there to work around shortcomings of the x/net package.

Reply:
Can you link me to the code that's working around shortcomings that we'll be able to remove?

Reply:
https://github.com/gravitational/teleport/blob/master/lib/web/terminal.go#L442 This among other bits.
It's all related to the manual ping loops we've employed since `x/net/websocket` never implemented ping keepalive. Most sane implementations handle this internally.